packages/twenty-server/src/workspace/workspace-query-runner/factories/arg-setters.factory.ts

```diff
@@ -0,0 +1,117 @@
+import { Injectable } from '@nestjs/common';
+
+import { FieldMetadataInterface } from 'src/metadata/field-metadata/interfaces/field-metadata.interface';
+import { WorkspaceQueryRunnerOptions } from 'src/workspace/workspace-query-runner/interfaces/query-runner-option.interface';
+import { ObjectMetadataInterface } from 'src/metadata/field-metadata/interfaces/object-metadata.interface';
+
+import { WorkspaceDataSourceService } from 'src/workspace/workspace-datasource/workspace-datasource.service';
+import { FieldMetadataType } from 'src/metadata/field-metadata/field-metadata.entity';
+
+@Injectable()
+export class ArgsSettersFactory {
```

Comment:
Not a big fan of the name since it does not "set" existing args but rather build new ones. But I don't know how I should call it.

Reply:
I agree, maybe we can just call it `ArgFactory` for now ?
cc @charlesBochet

Reply:
renamed QueryRunnerArgFactory so we avoid mistakes between existing arg factories in the builder